### PR TITLE
Created Logistic_Regression.py as classification algo

### PR DESCRIPTION
**Pull request by NEERAJAP2001**
**PR for #21** 

1. Created a new file for the Logistic_Regression algorithm.

2. created a new directory(same as regression) for this job.

3. Added Tests

### CODE REVIEW

**Comment by NEERAJAP2001** [comment]
Travis says **ModuleNotFoundError: No module named 'datascientist.model.classification.skl'**  
I created that directory to store the logistic_regression.py file.

**Comment by sidhu1012** [comment]
> Travis says **ModuleNotFoundError: No module named 'datascientist.model.classification.skl'**
> I created that directory to store the logistic_regression.py file.

`pip install -e .`
`pip install -r requirements.txt`

Use these commands in terminal and push again.

File: datascientist/model/classification/skl/logistic_regression.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    if metrics == 'jaccard':
+        accuracy = jaccard_score(test[1], y_hat)
+   
+
+    if x_predict is None:
+        return (model_name, accuracy, None)
+
+    y_predict = model.predict(x_predict)
+    return (model_name, accuracy, y_predict)
+
```
Comment (sidhu1012) [inline datascientist/model/classification/skl/logistic_regression.py:36]: Remove blank line

File: datascientist/model/classification/skl/logistic_regression.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+def _logistic_regression(*, train, test,x_predict=None,metrics=None,intercept_scaling=1, class_weight=None, random_state=None, solver='lbfgs', max_iter=100, multi_class='auto', verbose=0, warm_start=False, n_jobs=None, l1_ratio=None):
+    """
+    For more info visit : 
+    https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression  
+    """
+
+    model= LogisticRegression(random_state=0)
```
Comment (sidhu1012) [inline datascientist/model/classification/skl/logistic_regression.py:16]: Consider all arguments here.

File: datascientist/model/classification/skl/logistic_regression.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (sidhu1012) [inline datascientist/model/classification/skl/logistic_regression.py:1]: Remove it.

**Comment by sidhu1012** [comment]
File path is wrong.

**Comment by NEERAJAP2001** [comment]
Not a Problem @sidhu1012 I'll Come With a New PR!!

**Pull request closed without merging**